tests/framework/installer/ceph_installer.go

```diff
@@ -112,29 +113,31 @@ func (h *CephInstaller) CreateCephOperator(namespace string) (err error) {
 }
 
 func (h *CephInstaller) startAdmissionController(namespace string) error {
-	err := h.k8shelper.CreateNamespace(namespace)
-	if err != nil {
-		if errors.IsAlreadyExists(err) {
-			logger.Warningf("Namespace %q already exists!!!", namespace)
-		} else {
-			return fmt.Errorf("failed to create namespace %q. %v", namespace, err)
+	if h.k8shelper.VersionAtLeast("v1.15.0") {
```

Comment:
nit: as a general coding style, it's nice to return immediately.
```suggestion
	if !h.k8shelper.VersionAtLeast("v1.15.0") {
	  logger.Infof("skipping starting the admission controller")
	  return nil
	}
```

Reply:
done